js/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "defaultazurecredentialadapter",
+  "version": "1.0.0",
+  "description": "1.0.0",
+  "main": "index.js",
+  "scripts": {
+    "build": "tsc",
+    "dev": "tsc --watch & nodemon dist",
+    "test": "tsc && mocha dist/*.spec.js",
+    "lint": "eslint src --ext ts",
+    "tsc": "tsc",
+    "start": "node dist/defaultAzureCredentialAdapter.js"
+  },
+  "dependencies": {
+    "@azure/arm-keyvault": "^1.2.1",
+    "@azure/identity": "^1.0.2",
+    "@azure/ms-rest-nodeauth": "^2.0.5",
+    "@types/mocha": "^7.0.2"
+  },
+  "devDependencies": {},
```

Comment:
`mocha` is missing here.

Reply:
Shouldn't mocha be in devDependencies, not dependencies?